pages/about/groups/task-forces/pdf-aam.md

```diff
@@ -0,0 +1,98 @@
+- [Web-Based Surveys (WBS)](https://www.w3.org/2002/09/wbs/159734/)
+
+These tools are used by the Task Force participants. For ways that non-participants can contribute to the ARIA Working Group, including filing comments on publications, see [ARIA Contribution](/about/groups/ariawg/contribute/).
+
+### Teleconferences
+
+The PDF-AAM Task Force meets the third Wednesday of the month, 1:00–1:30 PM Eastern Time. For more details, including the meeting time in local time zones, see [PDF-AAM Calendar](https://www.w3.org/groups/tf/pdf-aam/calendar).
+
+### Meeting Minutes
+
+[Minutes from previous meetings](https://www.w3.org/WAI/ARIA/task-forces/pdf-aam/minutes) are available.
+
+### Mailing Lists
+
+The PDF-AAM Task Force uses the following mailing list for email discussion: [public-pdf-aam@w3.org Mail Archives](http://lists.w3.org/Archives/Public/public-pdf-aam/). 
+
+Participants are automatically added to the mailing list when they become a participant of the Task Force.
+
+## Current Work
+
+For the current PDF Accessibility API mappings, see the [Editor’s Draft](https://w3c.github.io/pdf-aam/).
+
+## Publications
+
+The PDF-AAM Task Force is primarily developing [PDF Accessibility API Mappings 1.0](https://w3c.github.io/pdf-aam/). 
+
+See also the [PDF-AAM GitHub issue tracking repository](https://github.com/w3c/pdf-aam/).
+
+## How to Comment, Contribute, and Participate
+
+To join the PDF-AAM Task Force, you must first join the [ARIA Working Group](/about/groups/apawg/). Participants are expected to [actively contribute](/about/groups/task-forces/pdf-aam/work-statement/#participation) to the work of the Task Force. 
+
+If you are interested in participating, email the PDF-AAM Task Force facilitators, [Duff Johnson and Valerie Young](mailto:duff.johnson@pdfa.org,spectranaut@igalia.com?subject=PDF-AAM%20Task%20Force%20Enquiry&cc=wai@w3.org), and include a little bit about what you’re interested in and how you think that you may be able to contribute to the Task Force. Then follow the [ARIA Working Group participation](/about/groups/ariawg/participate/) procedures to join the Working Group. Once you have joined the Working Group, email the group’s W3C Staff Contact, [Daniel Montalvo](mailto:dmontalvo@w3.org), and ask them to add you to the Task Force.
+
+For ways you can contribute to the work without joining the Task Force, see [ARIA Contribution](/about/groups/ariawg/contribute/). You can also contribute directly to documents under development in the [PDF-AAM GitHub source repository](https://github.com/w3c/pdf-aam/).
+
+[Current participants in the PDF-AAM Task Force](https://www.w3.org/groups/tf/pdf-aam/participants/).
+
+## Administrative Information
+
+The PDF-AAM Task Force is a task force of the [Accessible Rich Internet Applications (ARIA) Working Group](/about/groups/ariawg/). It assists this Working Group to develop [PDF-AAM](http://w3c.github.io/pdf-aam/) and keep it in sync with WAI-ARIA.
```

Comment:
Suggest removing this sentence as it is duplicated above.

Reply:
Done.